build/ps-entry.sh

```diff
@@ -424,14 +432,19 @@ if [[ $originalArgOne == mongo* ]]; then
 			# if --config arg is present, try to get tlsMode from it
 			if _parse_config "${mongodHackedArgs[@]}"; then
 				tlsMode=$(jq -r '.net.tls.mode // "preferSSL"' "${jsonConfigFile}")
+
+				if [ -z "$tlsMode" ]; then
+					tlsMode='preferTLS'
+				fi
 			fi
 			_mongod_hack_ensure_arg_val --sslMode "${tlsMode}" "${mongodHackedArgs[@]}"
 		fi
 	fi
 
-	MONGODB_VERSION=$(mongod --version | head -1 | awk '{print $3}' | awk -F'.' '{print $1"."$2}')
+	# MONGODB_VERSION=$(mongod --version | head -1 | awk '{print $3}' | awk -F'.' '{print $1"."$2}')
```

Comment:
why do we have this commented?

Reply:
Forgot to remove it, we now get MONGODB_VERSION earlier in the script.